### PR TITLE
chore: Update RateLimits.tsx

### PR DESCRIPTION
**Pull request by mansueli**
Make it clearer

## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

image

There's already hour mentioned in the other side.

## What kind of change does this PR introduce?

Bug fix, feature, docs update, ...

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

**Review by J0 — APPROVED** [review]
Thanks!

File: apps/studio/components/interfaces/Auth/RateLimits/RateLimits.tsx [hunk 1]
```diff
@@ -408,8 +408,8 @@ const RateLimits = () => {
                   <FormSectionLabel
                     description={
                       <p className="text-foreground-light text-sm">
-                        Number of sign up and sign-in requests that can be made per hour per IP
-                        address (excludes anonymous users).
+                        Number of sign up and sign-in requests that can be made per IP address
+                        (excludes anonymous users).
```
Comment (alaister) [inline apps/studio/components/interfaces/Auth/RateLimits/RateLimits.tsx:412]: We don't say anywhere that this is in a 5 minute interval though right?
```suggestion
                        Number of sign up and sign-in requests that can be made in a 5 minute interval per IP address
                        (excludes anonymous users).
```

Comment (mansueli) [reply apps/studio/components/interfaces/Auth/RateLimits/RateLimits.tsx:412]: But the hint below the editable field shows the per hour rate. I am not sure where you are getting this 5 minutes from. 🤔

Comment (joshenlim) [reply apps/studio/components/interfaces/Auth/RateLimits/RateLimits.tsx:412]: just chatted with Joel - our docs need some updating, alaister is right here RE the 5 minute interval although we currently don't mention about it in our docs

Comment (J0) [reply apps/studio/components/interfaces/Auth/RateLimits/RateLimits.tsx:412]: Updated here: https://github.com/supabase/supabase/pull/29176/

**Review by ivasilov — APPROVED** [review]

**Pull request merged**